subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1370,10 +1370,37 @@ void ll_rx_mem_release(void **node_rx)
 		{
 			struct node_rx_sync *se =
 				(void *)((struct node_rx_pdu *)rx_free)->pdu;
+			uint8_t status = se->status;
+
+			/* Below status codes use node_rx_sync_estab, hence
```

Comment:
This comment is a bit unclear for me. Maybe: 
`/* Below status codes are only allowed codes for node_rx_sync_estab, hence release of node memory and optional sync cleanup. */`

Reply:
Updated.